api/control.proto

```diff
@@ -29,6 +41,9 @@ service Control {
 	rpc ListTasks(ListTasksRequest) returns (ListTasksResponse) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager" };
 	};
+	rpc Attach(stream TaskExecStream) returns (stream TaskExecStream) {
+		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager"};
+	}
```

Comment:
Also, since this is executing a task, not attaching to the existing task, you should call this `Exec` and formally pass the command an arguments to be exec'd.